### PR TITLE
Update background color of select panel

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes [44](https://github.com/pxblue/angular-themes/issues/44) .

#### Changes proposed in this Pull Request:
- Add background color for mat-select-panel

#### Screenshots: 
This is SC of Angular design pattern
Screenshot 2021-11-30 at 4 12 19 PM
 
Figma Design link:
https://www.figma.com/file/ZzZJSoGjeKdrYct57EMPS0/Design-Patterns-Mono-Repo-Redesign?node-id=1339%3A182024

### CODE REVIEW

**Comment by emclaug2** [comment]
Could you please increase the version number for the package & add a changelog for this as well?

**Comment by ektasawant** [comment]
> Could you please increase the version number for the package & add a changelog for this as well?

Done please review .

**Comment by emclaug2** [comment]
Looking at the figma, this color can be translated to black[500] @ .05 opacity. 

image

So instead of hard-coding the color, we can use the palette here: `rbga(map-get($blui-black, 500), .05)`.

Realizing this, could you also update these items as well ?
image

**Comment by ektasawant** [comment]
> Looking at the figma, this color can be translated to black[500] @ .05 opacity.
> 
> Figma: image
> 
> PX Blue Colors Page: image
> 
> So instead of hard-coding the color, we can use the palette here: `rbga(map-get($blui-black, 500), .05)`.
> 
> Realizing this, could you also update these items in `theme.scss` as well ? image

Done please review.

File: _blueTheme.scss [hunk 1]
```diff
@@ -181,12 +181,12 @@
         avatarBackground: map-get($blui-gray, 500),
         avatarDisabledColor: map-get($blui-white, 50),
         avatarDisabledBackground: map-get($blui-gray, 500),
-        borderColor: rgba(66, 78, 84, 0.12),
+        borderColor: rgba(map-get($blui-black, 500), .12),
     );
     $chipOutlineDisabled: (
-        color: rgba(66, 78, 84, 0.3),
-        borderColor: rgba(66, 78, 84, 0.12),
-        removeIconColor: rgba(66, 78, 84, 0.3),
```
Comment (emclaug2) [inline _blueTheme.scss:189]: Looks like this should be .3 opacity.

File: _blueTheme.scss [hunk 2]
```diff
@@ -181,12 +181,12 @@
         avatarBackground: map-get($blui-gray, 500),
         avatarDisabledColor: map-get($blui-white, 50),
         avatarDisabledBackground: map-get($blui-gray, 500),
-        borderColor: rgba(66, 78, 84, 0.12),
+        borderColor: rgba(map-get($blui-black, 500), .12),
     );
     $chipOutlineDisabled: (
-        color: rgba(66, 78, 84, 0.3),
```
Comment (emclaug2) [inline _blueTheme.scss:187]: Same here.

File: _blueTheme.scss [hunk 3]
```diff
@@ -288,13 +288,13 @@
         outline: (
             text: map-get($blui-black, 500),
             background: transparent,
-            backgroundHover: rgba(66, 78, 84, 0.05),
-            border: rgba(66, 78, 84, 0.3),
```
Comment (emclaug2) [inline _blueTheme.scss:292]: .3, not .03 opacity

File: theme.scss [hunk 4]
```diff
@@ -59,11 +59,11 @@ $material-theme: mat-light-theme($material-theme-primary, $material-theme-accent
                 secondary-text: map-get($blui-gray, 500),
                 hint-text: map-get($blui-gray, 500),
                 base: map-get($blui-darkBlack, 900),
-                divider: rgba(66, 78, 84, 0.12),
+                divider: rgba(map-get($blui-black, 500), .12),
                 dividers: $dark-dividers,
-                disabled: rgba(66, 78, 84, 0.3),
+                disabled: rgba(map-get($blui-black, 500), .03),
                 disabled-button: rgba(black, 0.26),
-                disabled-text: rgba(66, 78, 84, 0.3),
```
Comment (emclaug2) [inline theme.scss:66]: .3 opacity, not .03

Thanks for going through the themes and finding all these missed palette refs.

Comment (ektasawant) [reply theme.scss:66]: Sure let me update.

Comment (ektasawant) [reply theme.scss:66]: Hi @emclaug2 , please review.

**Review by emclaug2 — APPROVED** [review]

**Comment by huayunh** [comment]
In the design world, the active state in a menu item typically looks like the screen below. Read more at https://material.io/components/menus#usage 

image

**Comment by emclaug2** [comment]
@huayunh We won't be able to do something like that via themes, it'd require a new component.  There are mixed examples in example usage section: https://material.angular.io/components/select/examples

Sometimes active item bg color matches hover bg color, other times active item bg color is darker than hover color.  We used to have the second option.

Should we revert the mat-select style changes?

**Comment by ektasawant** [comment]
> @huayunh We won't be able to do something like that via themes, it'd require a new component. There are mixed examples in example usage section: https://material.angular.io/components/select/examples
> 
> Sometimes active item bg color matches hover bg color, other times active item bg color is darker than hover color. We used to have the second scenario.
> 
> Should we revert the mat-select style changes?

Hi @huayunh , do we need to revert this changes?

**Comment by huayunh** [comment]
Visually, what you changed there is fine. I find it consistent with what ng mat has to offer for its out of the box styles.

**Comment by ektasawant** [comment]
> Visually, what you changed there is fine. I find it consistent with what ng mat has to offer for its out of the box styles.

Shall we add this same background color changes in React and React Native framework ?

**Comment by huayunh** [comment]
Do React and React Native have an active state like that?

**Comment by emclaug2** [comment]
This is React's, I'm going to get RN's too: 

React

**Pull request merged**